Streetcode/Streetcode.BLL/MediatR/Streetcode/Streetcode/GetPageMainPage/GetPageOfStreetcodesMainPageHandler.cs

```diff
@@ -45,7 +46,15 @@ public async Task<Result<IEnumerable<StreetcodeMainPageDTO>>> Handle(GetPageOfSt
 
                 var streetcodesPaginated = streetcodes.Paginate(request.page, request.pageSize);
 
-                return Result.Ok(_mapper.Map<IEnumerable<StreetcodeMainPageDTO>>(streetcodesPaginated));
+                var shuffledStreetcodes = streetcodes.OrderBy(sc =>
```

Comment:
Why do we need `streetcodesPaginated` variable if we don't use it? Is it worth using it here?

Reply:
Oops! My bad, I accidentally used streetcodes fro shuffling instead of streetcodePaginated